addons/hr_expense/tests/test_expenses_mail_import.py

```diff
@@ -212,3 +213,17 @@ def assertParsedValues(subject, currencies, exp_description, exp_amount, exp_pro
             self.product_a,
             self.company_data['currency'],
         )
+
+    def test_import_expense_from_mail_get_default_expense_sheet_values_errors(self):
+        # Make sure we get the expected UserError when trying to validate an expense with no product
+        message = {
+            'message_id': "the-world-is-a-ghetto",
```

Comment:
Not a fan of putting 'ghetto' in the source code in community, but since it's already present in other parts... why not (ideally I would have put a comment to the referred song but no need to change)